ons_alpha/methodologies/models.py

```diff
@@ -0,0 +1,95 @@
+from functools import cached_property
+
+from django.db import models
+from wagtail.admin.panels import (
+    FieldPanel,
+    MultiFieldPanel,
+    MultipleChooserPanel,
+    ObjectList,
+    TabbedInterface,
+)
+from wagtail.search import index
+
+from ons_alpha.bundles.models import BundledPageMixin
+from ons_alpha.core.blocks.stream_blocks import CoreStoryBlock
+from ons_alpha.core.models.base import BasePage
+from ons_alpha.taxonomy.forms import PageWithTopicsAdminForm
+from ons_alpha.utils.fields import StreamField
+
+
+class MethodologyPage(BundledPageMixin, BasePage):
+    base_form_class = PageWithTopicsAdminForm
+    template = "templates/pages/methodology_page.html"
+    parent_page_types = ["topics.TopicPage"]
+    subpage_types = []
+
+    summary = models.TextField()
+    last_revised_date = models.DateField(blank=True, null=True)
+    how_it_was_revised = models.CharField(blank=True, max_length=255)
```

Comment:
Where this is rendered looks odd but okay for the poc.

Reply:
I went back and forth on where to display it. Will move it to the main content area